src/openrct2/paint/tile_element/Segment.h

```diff
@@ -27,7 +28,9 @@ enum class PaintSegment : uint16_t
     topLeftSide = 7,
     centre = 8,
 };
+constexpr uint16_t kSegmentsNone = std::numeric_limits<uint16_t>::max();
```

Comment:
Since a segment is blocked when a bit is set, the name `kSegmentsNone`  implies 0, not 0xFFFF.